### PR TITLE
fix: LEAP-930: Set model as prelabeler

### PR DESCRIPTION
**Pull request by hlomzik**
This code sets a model as a default prelabeler in case no other option was selected. This is done to keep it consistent with how offline predictions are used when uploaded (they get selected).

That's basically a #5695 fixed for Follow Merge

### PR fulfills these requirements
- [x] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
- [ ] Product design
- [x] Backend (Database)
- [x] Backend (API)
- [ ] Frontend

### Describe the reason for change
Label Stream and other flows have some issues when model is added but not selected because of discrepancies in logic.

#### What alternative approaches were there?
We could change the way model is added and handled in settings, but autoselect is small and precise fix.

### Does this PR introduce a breaking change?
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
- [ ] e2e
- [x] integration
- [ ] unit

### CODE REVIEW

File: label_studio/ml/api.py [hunk 1]
```diff
@@ -88,6 +88,15 @@ def perform_create(self, serializer):
         ml_backend = serializer.save()
         ml_backend.update_state()
 
+        project = ml_backend.project
+
+        # In case we are adding the model, let's set it as the default
+        # to obtain predictions. This approach is consistent with uploading
+        # offline predictions, which would be set automatically.
+        if project.show_collab_predictions and (project.model_version is None or project.model_version == ''):
```
Comment (jombooth) [inline label_studio/ml/api.py]: this latter conjunct is the same as `not project.model_version`

File: scripts/update_ml_tutorials.py [hunk 2]
```diff
@@ -25,10 +25,10 @@
```
Comment (jombooth) [inline scripts/update_ml_tutorials.py:25]: Did anything relevant to the diff actually change in this PR? If not, please revert the changes to this file.

File: label_studio/projects/serializers.py [hunk 3]
```diff
@@ -94,6 +94,7 @@ def to_internal_value(self, data):
         # FIXME: remake this logic with start_training_on_annotation_update
         initial_data = data
         data = super().to_internal_value(data)
+
```
Comment (jombooth) [inline label_studio/projects/serializers.py:97]: super nit: as long as we're making small changes, let's remove the blank line

File: label_studio/tests/ml/test_api.py [hunk 4]
```diff
@@ -22,13 +26,72 @@ def mock_gethostbyname(mocker):
     mocker.patch('socket.gethostbyname', return_value='321.21.21.21')
 
 
+@pytest.mark.django_db
```
Comment (jombooth) [inline label_studio/tests/ml/test_api.py:29]: great tests, TY

**Comment by jombooth** [comment]
A few small requests here

File: scripts/update_ml_tutorials.py (hunk 2 above)
Comment (jombooth) [reply scripts/update_ml_tutorials.py:25]: uhm, never mind, I guess the our linter picks up this script? I see the same changes on https://github.com/HumanSignal/label-studio/pull/5686/commits/971b8b58e20a8da7c7135b554fd133e73055db90, and in any case nothing appears to be broken

**Review by jombooth — APPROVED** [review]
Going to approve because no blocking feedback. I'm not sure what happened with that script having formatting changes.

File: label_studio/ml/api.py (hunk 1 above)
Comment (hlomzik) [reply label_studio/ml/api.py]: yeah, I also thought so!

Comment (jombooth) [reply label_studio/ml/api.py]:
```suggestion
        if project.show_collab_predictions and not project.model_version:
```

File: label_studio/projects/serializers.py (hunk 3 above)
Comment (hlomzik) [reply label_studio/projects/serializers.py:97]: we have such separation of vars and code in frontend maybe it's the same? I'm going to resolve this anyway to merge PR
